frontend/userFeaturePortal-frontend/src/app/components/dji store/dji.store.component.scss

```diff
@@ -0,0 +1,59 @@
+.main-card {
+  height: 40000px;
+  background: #666666;
+}
+
+.Drones-text {
+  font-family: historical-forms;
+  font-size: 100px;
+  position: absolute;
+  top: 0.1%;
+  left: 40%;
+}
+
+.mini {
+  scale: 125%;
+  background-color: #666666;
+  color: #666666;
+  margin-right: 575px;
+  margin-left: 5px;
+  margin-top: 250px;
+}
+
+.mavic {
+  color: #666666;
+  background-color: #666666;
+  scale: 162%;
+}
+
+.air {
+  color: #666666;
+  background-color: #666666;
+  scale: 104.5%;
+  margin-bottom: 300px;
+}
+
+.fpv {
+  color: #666666;
+  background-color: #666666;
+  scale: 210%;
+  margin-bottom: 350px;
+}
+
+.phantom {
+  color: #666666;
+  background-color: #666666;
+  scale: 115%;
+}
+
+.inspier {
+  color: #666666;
+  background-color: #666666;
+  scale: 91%;
+}
+
+.thermal {
```

Comment:
Die Klassen sind alle nichtssagend, bitte funktionalität in den Klassen beschreiben